pipeline/sources/authorities/getty/mapper.py

```diff
@@ -521,14 +521,17 @@ def transform(self, record, rectype, reference=False):
             brdrs2 = rec.get('broader', [])
             if type(brdrs2) != list:
                 brdrs2 = [brdrs2]       
-            brdrs.extend(brdrs2)        
+            brdrs.extend(brdrs2)
             for br in brdrs:
                 if type(br) == str:
                     br = {'id': br, '_label': ""}
                 lbl = br.get("_label", "")
                 if type(lbl) == dict:
                     lbl = lbl['@value']
-                top.part_of = model.Place(ident=br['id'], label=lbl)
+                if "classified_as" in br:
```

Comment:
I think this should test if there are any classified_as across the `brdrs` -- we don't want to lose all of them if none have a classification.

Reply:
@azaroth42 do you think it would be better to go the route I originally intended and acquire the referenced rec and check for the `former political entity` classification?

Reply:
I think this approach is fine -- there could be other reasons to have a preferred and then other non-preferred too that this would catch. We just don't want to throw out all of them.